packages/eas-cli/src/build/metadata.ts

```diff
@@ -30,8 +31,9 @@ export async function collectMetadataAsync<T extends Platform>(
     workflow: ctx.workflow,
     credentialsSource: ctx.buildProfile.credentialsSource,
     sdkVersion: ctx.exp.sdkVersion,
-    runtimeVersion: runtimeMetadata?.runtimeVersion,
-    fingerprintSource: runtimeMetadata?.fingerprintSource,
+    runtimeVersion: runtimeAndFingerprintMetadata?.runtimeVersion,
+    fingerprintHash: runtimeAndFingerprintMetadata?.fingerprintHash,
```

Comment:
I believe we also need to add it to metadata GQL input and our types in WWW.

Example PR that did it from the past: https://github.com/expo/universe/pull/15517/files#diff-e2044eb5ede366c64cbc5ee72521add4fc5e641b2c841c79f9e1e129aaaef363

Reply:
Merged in https://github.com/expo/universe/pull/17353 :)